modules/shortcodes/gist.php

```diff
@@ -156,6 +166,12 @@ function github_gist_shortcode( $atts, $content = '' ) {
 		$file = rawurlencode( $file );
 	}
 
+	// Set the tab size, allowing attributes to override the query string.
+	$tab_size = $gist_info['ts'];
+	if ( ! empty( $atts['ts'] ) ) {
```

Comment:
I decided to allow the attributes to override the query string to allow for theme filtering via [`shortcode_atts_{$shortcode}`](https://developer.wordpress.org/reference/hooks/shortcode_atts_shortcode/) and similar.

Reply:
Should we add some default attributes in `shortcode_atts` at the top of `github_gist_shortcode` to allow for this to happen?

Reply:
The code already sets the defaults in `jetpack_gist_get_shortcode_id`. Do you feel that it needs to have that set additionally?